integration/docker/Dockerfile.fuse

```diff
@@ -18,6 +18,7 @@
 FROM alpine:3.10.2 AS extractor
 # Note that downloads for *-SNAPSHOT tarballs are not available
 ARG ALLUXIO_TARBALL=http://downloads.alluxio.io/downloads/files/2.4.0-SNAPSHOT/alluxio-2.4.0-SNAPSHOT-bin.tar.gz
+ARG ENABLE_DYNAMIC_USER=false
```

Comment:
Please add the same comment and printout as the Dockerfile.

Reply:
fixed